templates/company/about.html

```diff
@@ -0,0 +1,464 @@
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-start-medium-3 col-medium-4 col-start-large-4 col-9">
+        <div class="p-cta-block">
+          <a href="https://ubuntu.com/engage?resource=case+study">Browse customer stories&nbsp;&rsaquo;</a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Where to find us</h2>
+      </div>
+      <div class="col">
+        <p>
+          With corporate hubs and a global team across APAC, EMEA and North America, we deliver the reliable support you need to innovate with confidence.
+        </p>
+        <div class="p-cta-block">
+          <a href="/contact-us">Contact our team&nbsp;&rsaquo;</a>
+        </div>
+      </div>
+    </div>
+    <!-- map goes here -->
+    <div class="row">
+      <figure>
+        <div id="sprint-map"></div>
+        <div class="row">
+          <figcaption class="col-6 col-medium-3 col-start-large-7 u-text--muted">Where to find us</figcaption>
+        </div>
+      </figure>
+    </div>
+  </section>
+
+  <section class="p-section--deep">
+    <div class="row--25-75">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>See our mission in action</h2>
+      </div>
+      <div class="col">
+        <div class="row">
+          <div class="col-3">
+            <h3 class="p-heading--5">See what our team is up to</h3>
+          </div>
+          <div class="col-6">
+            <p>Get the latest news on our company, products and services.</p>
+            <div class="p-cta-block">
+              <a href="/press-centre">Visit our press center&nbsp;&rsaquo;</a>
+            </div>
+          </div>
+        </div>
+        <hr class="p-rule--muted" />
+        <div class="row">
+          <div class="col-3">
+            <h3 class="p-heading--5">Become a partner</h3>
+          </div>
+          <div class="col-6">
+            <p>Work with us to amplify the impact of open source.</p>
+            <div class="p-cta-block">
+              <a href="/partners">Discover our partner programmes&nbsp;&rsaquo;</a>
+            </div>
+          </div>
+        </div>
+        <hr class="p-rule--muted" />
+        <div class="row">
+          <div class="col-3">
+            <h3 class="p-heading--5">Get news from The Source</h3>
+          </div>
+          <div class="col-6">
+            <p>An opinionated and entertaining view on all things open source.</p>
+            <div class="p-cta-block">
+              <a href="https://www.linkedin.com/build-relation/newsletter-follow?entityUrn=7108409718422839296">Subscribe to our LinkedIn newsletter&nbsp;&rsaquo;</a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <script>
+    const sprintLocations = [
```

Comment:
The original map this is based on was for sprints, but this is for office locations. Would be good if we could be clear with variable names, but you would have to edit the script that generates this